vdr/vdr.go

```diff
@@ -54,9 +48,9 @@ import (
 type VDR struct {
 	Config            Config
 	store             types.Store
-	network           networkPkg.NetworkClient
+	network           network.Network
```

Comment:
`network = network.Network`, there might be some kind of anti-pattern in the implementation somewhere....

Reply:
Will fix later; https://github.com/nuts-foundation/nuts-node/issues/18